File: java/parser/src/main/java/gov/lbl/parser/domain/Modification.java
package gov.lbl.parser.domain;

public class Modification {
    private Class_modification class_modification;
    private Boolean equal;
    private Boolean colon_equal;
    private Expression expression;

    public Modification(Class_modification class_modification, Boolean equal, Boolean colon_equal, Expression expression) {
        this.class_modification = class_modification;
        this.equal = equal;
        this.colon_equal = colon_equal;
        this.expression = expression;
    }
}

File: jsParser/domain/Modification.js
class Modification {
  constructor (class_modification, equal, colon_equal, expression) {
    class_modification != null ? this.class_modification = class_modification : ''
    equal != null ? this.equal = equal : ''
    colon_equal != null ? this.colon_equal = colon_equal : ''
    expression != null ? this.expression = expression : ''
  }
}
exports.Modification = Modification
